modules/waf-regional/wafregional_ruleset8_csrf.tf

```diff
@@ -49,3 +61,32 @@ resource "aws_wafregional_size_constraint_set" "csrf_token_set" {
   }
 }
 
+resource "aws_wafregional_byte_match_set" "csrf_fetch_same_site" {
```

Comment:
@exequielrafaela There is a lot of magic going here :) -- I mean, I know CloudFront uses its own conventions to accomplish several goals, I just think it would nice to have a single line that references the documentation that explains where the values in the "byte_match_tuples" block come from (target_string, data, etc).

Reply:
@joekhoobyar Do you think that would be possible to attend @diego-ojeda-binbash request :point_up: before we merge this PR? Thanks in advance! :pray: